v1/pipeline/steps/cd/k8s-scale-step-info.yaml

```diff
@@ -3,48 +3,18 @@ allOf:
 - $ref: ../../common/step-spec-type.yaml
 - type: object
   required:
-  - instanceSelection
-  - workload
+    - instanceSelection
+    - workload
   properties:
-    delegateSelectors:
-      oneOf:
-      - type: array
-        items:
-          type: string
-      - type: string
-        pattern: (<\+.+>.*)
-        minLength: 1
-    instanceSelection:
+    delegates:
+      $ref: ../../common/delegate.yaml
+    instance_selection:
       $ref: instance-selection-wrapper.yaml
-    skipSteadyStateCheck:
+    skip_steady_state_check:
```

Comment:
At one place we are following the skip: then internal keys that we want to skip. But here we are flattening the skip_keys.
Same for `skip_dry_run` in k8s-rolling-step-info.yaml
And at another place its 
skip:
  dry_run: 
  steady_state_check: 

We should follow one pattern.

Reply:
done